test/object_test.rb

```diff
@@ -57,4 +58,19 @@ class ObjectTest < MiniTest::Spec
       _(source.album.songs[0].title).must_equal 1
     end
   end
-end
\ No newline at end of file
+
+  describe ".format_engine" do
+    let(:decorator) do
+      class DecoratedObject < Representable::Decorator
+        include Representable::Object
+
+        property :city, as: :city_name
+      end
+
+      DecoratedObject
+    end
+
+    it { decorator.must_equal(ObjectTest::DecoratedObject) }
+    it { decorator.format_engine.must_equal(Representable::Object) }
```

Comment:
@dmuneras We should verify if `representer` renders correctly using `to_object`. You can check out the other test cases in this file for example.

Also, I don't think we need to test `.format_engine` as it's an internal API (as long as the `representer` generates correct result).